src/index.html

```diff
@@ -11,12 +11,102 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header class="header">
+      <img
+        src="images/logo.png"
+        alt="Moyo logo"
+      />
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav__link is-active"
+              href="#section-1"
+            >
+              APPLE
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#section-2"
+            >
+              SAMSUNG
+            </a>
+          </li>
```

Comment:
Add empty lines between multiline sibling blocks of HTML to add some 'air' and simplify reading. (HTML Formatting)